conftest.py

```
@@ -155,6 +173,17 @@ def pytest_sessionfinish(session, exitstatus):
     """Clean up at the end of the test session."""
     cleanup_test_artifacts()
 
+    # Clean up the custom pytest temp directory
+    project_root = Path(__file__).parent
+    custom_temp = project_root / ".pytest_tmp"
+
+    if custom_temp.exists():
+        try:
+            shutil.rmtree(custom_temp)
+            print(f"\nCleaned up pytest temp directory: {custom_temp}")
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)